package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "truex-shared",
-  "version": "1.6.13",
+  "version": "1.7.1",
```

Comment:
@addisonm can you use the 1.7.3 version and tag going forward
i need to make another intervening truex-shared fix for automation that i hope to merge today
so i will be taking the 1.7.1 tag
you will then need to merge that in to your branch first so that the final tags are cumulative